src/components/AudioFeatureSlider.tsx

```diff
@@ -0,0 +1,52 @@
+import React , { useState } from 'react';
+import Slider from '@material-ui/core/Slider';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core';
+
+const useStylesslider = makeStyles({
+    root: {
+      width: 300,
+    },
+  });
+export default function AudioFeatureSlider(props: {
+    feature_name: string,
+    feature_value: Number[],
```

Comment:
Please change variable and prop names to conform to camelCase

```suggestion
    featureName: string,
    featureValue: Number[],
```